lib/stripe.js

```diff
@@ -24,6 +24,9 @@ Stripe.USER_AGENT_SERIALIZED = null;
 var APP_INFO_PROPERTIES = ['name', 'version', 'url'];
 
 var exec = require('child_process').exec;
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var objectAssign = require('object-assign');
```

Comment:
No worries.  It hurts my soul that it's not at the top of the file.  My order of preference would actually be:

    var util = require('util');
    var objectAssign = require('object-assign');

    var exec = require('child_process').exec;
    var EventEmitter = require('events').EventEmitter;

But it sounds like this is what you're suggesting, right?

    var EventEmitter = require('events').EventEmitter;
    var exec = require('child_process').exec;
    var objectAssign = require('object-assign');
    var util = require('util');